packages/search/src/rules/noReferenceProjectActionRule.ts

```diff
@@ -1,88 +1,47 @@
+import { CustomActionModel } from '@toddledev/core/dist/component/component.types'
 import { ToddleComponent } from '@toddledev/core/dist/component/ToddleComponent'
-import { Formula } from '@toddledev/core/dist/formula/formula'
-import { isToddleFormula } from '@toddledev/core/dist/formula/formulaTypes'
 import type { Rule } from '../types'
 
-export const noReferenceProjectFormulaRule: Rule<void> = {
```

Comment:
What happened to the `noReferenceProjectFormulaRule` 🤔 did you duplicate or change the rule?

Reply:
The `noReferenceProjectActionRule` was a copy of the `noReferenceProjectFormulaRule` 🤷‍♂️ So it appears we just never finished the rule for unused actions...